idaplugin/rematch/network.py

```diff
@@ -130,8 +135,8 @@ def get_headers(token, json):
   if json:
     headers['Accept'] = 'application/json, text/html, */*'
     headers['Content-Type'] = 'application/json'
-  if token is None and 'token' in config and config['token']:
-    token = config['token']
+  if token is None and token in config['login']:
```

Comment:
`token in config['login']` is funny, should be `'token' in config['login'] I think.